validation: allow checks to be focussed to a single chain (#154)
ethereum-optimism/superchain-registry: CONTRIBUTING.md

- a `stateHash` attribute: to omit a large state (e.g. for networks with a re-genesis or migration history).
  Nodes can load the genesis block header, and state-sync to complete the node initialization.

### 4. Run tests locally

> Run the following command to run our validation checks, only on the chain you added:

Reviewer: nit
```suggestion
Run the following command to run the registry's validation checks, for only the chain you added:
```

```
cd validation
go test ./... -v -focus=true -chain-id=<your-chain-id-here>

> ```

Author: This is how you can now focus the validation checks to a single chain.

> go test ./... -v -focus-chain-id=<your-chain-id-here>

Reviewer 2: Could we consider ensuring that all per-chain tests use the test name & use the native `-run=regex` or does that not meet all of our requirements for this flag?

Author: Good suggestion. I think that could work, and would probably require less boiler plate code and a similar amount of diligence writing new tests. And it may allow focussing by name as well as by chain id, which would be nice.
